taos-query/src/common/raw/mod.rs

```diff
@@ -400,11 +400,7 @@ impl RawBlock {
 
                     data_lengths[i] = *length * rows as u32;
                 }
-                Ty::Decimal | Ty::Decimal64 => unimplemented!("decimal type not supported"),
-                Ty::VarBinary => todo!(),
-                Ty::Blob => todo!(),
-                Ty::MediumBlob => todo!(),
-                Ty::Geometry => todo!(),
+                _ => todo!(),
```

Comment:
Empty `todo!()` will be disabled, explain what to do here for better error message